### PR TITLE
update influence computation for PAD Factory

### PR DESCRIPTION
**Pull request by yads**
* include Core 2.0 PAD Campaign

### CODE REVIEW

**Comment by yads** [comment]
Fix for #138

File: src/AppBundle/Resources/public/js/nrdb.js [hunk 1]
```diff
@@ -566,7 +566,7 @@ function get_influence_cost_of_card_in_deck(card) {
             }
         } else if (card.code === '10038') {
             // PAD Factory: 3 PAD Campaign
-            if (count_card_copies(NRDB.data.cards.find({indeck: {'$gt': 0}, code: '01109'})) === 3) {
+            if (count_card_copies(NRDB.data.cards.find({indeck: {'$gt': 0}, code: '01109'})) === 3 || count_card_copies(NRDB.data.cards.find({indeck: {'$gt': 0}, code: '20128'})) === 3) {
```
Comment (alsciende) [inline src/AppBundle/Resources/public/js/nrdb.js]: You can use `NRDB.data.cards.find({indeck: {'$gt': 0}, code: ['01109','20128']})`. That way you'll only do 1 query to the db and count both Pad Campaign together.

Comment (yads) [reply src/AppBundle/Resources/public/js/nrdb.js]: I was wondering if there was a shortcut like that. Thanks, I'll update and push up.

**Comment by alsciende** [comment]
Thank you!

**Pull request merged**